RPIPs/RPIP-46.md

```diff
@@ -29,91 +29,121 @@ This RPIP is part of a set of proposals motivated by a desire to rework Rocket P
-   2. `voter_share`: each NO receives a share of revenue based on the vote-eligible RPL staked to their megapool. The overall voter share of revenue is based on the setting, and each NO receives a proportion of that based on `vote_eligible_RPL_in_their_megapool/total_vote_eligible_RPL_in_megapools`.
-   3. `surplus_share -  node_operator_commission_share_council_adder`: this share of revenue is used to distribute revenue beyond that used for protocol operation (such as the shares above)
-2. `reth_commission` SHALL be defined as the sum of `node_operator_commission_share`, `voter_share`, and `surplus_share`
+   2. `voter_share -  node_operator_commission_share_council_adder`: each NO receives a share of revenue based on the vote-eligible RPL staked to their megapool. The overall voter share of revenue is based on the setting, and each NO receives a proportion of that based on `vote_eligible_RPL_in_their_megapool/total_vote_eligible_RPL_in_megapools`.
+2. `reth_commission` SHALL be defined as the sum of `node_operator_commission_share`, and `voter_share`
 3. `reth_share` SHALL be defined as `100% - reth_commission`
 4. Distributions of revenue from borrowed ETH MUST respect the defined shares
    1. If shares change between claims, distributions MUST make an effort to account for the different values. For example, a distribution could use a duration-weighted average share. Approximations MAY be used where they significantly reduce complexity and/or costs.
    2. Legacy minipools are an exception and SHALL continue to support earlier distribution methodologies 
-5. `node_operator_commission_share`, `node_operator_commission_share_council_adder`, and `surplus_share`, SHALL be updateable by an address in the `allowlisted_controllers` array
+5. `node_operator_commission_share`, `node_operator_commission_share_council_adder`, and `voter_share`, SHALL be updateable by an address in the `allowlisted_controllers` array
    1. This functionality SHALL not be used without a separate pDAO vote to enable a controller and add it to the list
 6. The `node_operator_commission_share_council_adder` setting SHALL only allow values where:
    1. 0% ≤ `node_operator_commission_share_council_adder` ≤ `max_node_operator_commission_share_council_adder` 
-   2. `node_operator_commission_share_council_adder` ≤ `surplus_share`
+   2. `node_operator_commission_share_council_adder` ≤ `voter_share`
 7. The `node_operator_commission_share_council_adder` setting SHALL be controllable by the security council without requiring a delay
 8. The security council SHOULD increment `node_operator_commission_share_council_adder` by 0.5% if the deposit pool is over half-full for the majority of a 2-week period with a constant `node_operator_commission_share + node_operator_commission_share_council_adder`
```

Comment:
Is the team confirmed happy / accepting of this one? Last I saw @darrenlangley was against modifying it so frequently.

Reply:
![image](https://github.com/user-attachments/assets/01f08da2-91b6-4e03-99cf-785bb2319dbf)
Commented on it when in PR. The comment is focused on complexity etc, but didn't seem significantly against. My take was "not a fan, but accepting", but potentially read too much into it.

Reply:
I was commenting strictly from a saved dev effort perspective.

I am very worried about the security council being overreactive to short-term demand spikes. 

> The security council SHOULD increment `node_operator_commission_share_council_adder` by 0.5% if the deposit pool is over half-full for the majority of a 2-week period
This to me is crazy. 

I don't have a problem with delegating some of this power to save on pDAO voting and even being more responsive, we do not know how Saturn will react to demand. In my opinion before this is enabled we should have a separate discussion about how it is used - not bake it in now.

Reply:
I should say that we are fine to include the functionality, as it allows us to adapt to uncertainty. I just have the concerns above.